src/enrichment_tables/file.rs

```diff
@@ -114,16 +176,34 @@ impl File {
         self.headers.iter().position(|header| header == col)
     }
 
-    fn row_equals(&self, condition: &[Condition], row: &[String]) -> bool {
+    /// Currently all matches are case insensitive.
+    /// TODO We want to add a configuration option to allow for case sensitive searches. This will
```

Comment:
Do we have a follow-up issue / item on the check list on the enrichment table issue for this?

Reply:
#8815 They are all kind of bunched together a bit..